controls/radwindow/features/window-icon.md

```diff
@@ -10,15 +10,17 @@ position: 4
 
 # Window Icon
 
-__RadWindow__ allows you to display a custom icon in its top-left corner. To specify the icon you can use either the __Icon__ or the __IconTemplate__ properties.
+`RadWindow` allows you to display a custom icon in its top-left corner. To specify the icon you can use either the `Icon` or the `IconTemplate` properties.
 
-If you have a window-specific icon, use the __Icon__ property.
+## Icon Property
 
->tipAs the __Icon__ property is of type object you can set it to any control that you like. If you want to have a more complex icon content that consists of more than one control, be sure to wrap them inside a layout control and pass the layout control as content.
+If you have a window-specific icon, use the `Icon` property.
 
->__RadWindow__ is declared and opened from the code behind by default. The only way to use the __RadWindow__ as a visual element in XAML is when it represents the entire UserControl. To learn more about that read [Use RadWindow as User Control]({%slug radwindow-how-to-use-radwindow-as-user-control%}) article.
+>tip As the Icon property is of type `object` you can set it to any control that you like. If you want to have a more complex icon content that consists of more than one control, be sure to wrap them inside a layout control and pass the layout control as content.

```

Comment:
"As the Icon property is of type `object` you can set it to any control that you like."

how come? What does it mean that you can set it to any control? This sounds like you can assign the property to any control. For example "<Border Icon="myIcon.png">". You can try with something like "...is of type `object` you can assign any object to it.". Also, note that the property is of type Object and not Control. This means that you can assign it to any object, not only to any control.

"If you want to have a more complex icon content that consists of more than one control, be sure to wrap them inside a layout control and pass the layout control as content."

The "layout control" is not very clear. What does it mean? Isn't any control, a layout control? There is no concept for this wording in WPF. Also, you can remove the "that consists of more than one control" part and leave only the "If you want to have a more complex layout.." part. You can tell something like "To replace the image with a custom visual, you can define any UIElement and assing it to the Content". However, I am not sure if this is true. Please double check it. I think that you need to replace also the IconTemplate, but I am not sure if this is valid for RadWindow.